torch/_torch_docs.py

```diff
@@ -3512,6 +3524,12 @@ def merge_dicts(*dicts):
     tensor([[True, False], [True, True]])
 """.format(**common_args))
 
+add_docstr(torch.less_equal, r"""
+less_equal(input, other, *, out=None) -> Tensor
+
+Alias for :func:`torch.le`.
```

Comment:
why do we describe the alias in this direction and not the other?

Reply:
You mean, why doesn't the `torch.le` docstring say that it has an alias, `torch.less_equal`? We could do that.

Reply:
right, that's what I was asking.  Which do we "prefer"